.Documentation/articles/innoactive-creator/step-inspector.md

```diff
@@ -34,8 +34,8 @@ Learn more about the [default behaviors](default-behaviors.md).
 
 ## Add and Remove Transitions
 
-To add a transition in the Step Inspector, click on the `Transitions` tab and the `Add Transition` button. It will point to the end of the chapter by default. To choose another destination, see [the article about Workflow Editor](workflow-editor.md#add-transitions).  
-You can remove a transition by clicking on the <img src="../images/step-inspector/icon_delete_dark.png" alt="Trash Can Icon" height="16px"/> button in the upper right corner of it.
+To add a transition in the Step Inspector, click on the `Transitions` tab and the `Add Transition` button. It will point to the end of the chapter by default. To choose another destination, see [the article about Workflow window](workflow-editor.md#add-transitions).  
```

Comment:
```suggestion
To add a transition in the Step Inspector, click on the `Transitions` tab and the `Add Transition` button. It will point to the end of the chapter by default. To choose another destination, see [the article](workflow-editor.md#add-transitions) about Workflow window.  
```

Reply:
Like above: I prefer the name as link